### PR TITLE
Add several Malayalam courses authored by `Yes Tech Media`, `Tintu Vlogger` and `Brototype Malayalam` 

### PR DESCRIPTION
**Pull request by GH05T-HUNTER5**
Update malyalam Course

### CODE REVIEW

**Comment by GH05T-HUNTER5** [comment]
Malayalam New Tutorial Playlist

File: courses/free-courses-ml.md [hunk 1]
```diff
@@ -1,33 +1,113 @@
...
+* [C](#c)
+* [C++](#cpp)
+* [Data Structure](#data-structure)
+* [Flutter](#flutter)
+* [Git](#git)
+* [HTML and CSS](html-and-css)
+* [Java](#java)
 * [JavaScript](#javascript)
+    * [React](#react)
+    * [NodeJS](#nodejs)
```
Comment (davorpa) [inline courses/free-courses-ml.md:15]: - Fix `node.js` title like in other listings
- Alphabetize sections

```suggestion
    * [Node.js](#nodejs)
    * [React](#react)
```

File: courses/free-courses-ml.md [hunk 2]
```diff
@@ -1,33 +1,113 @@
...
     * [Django](#django)
+    * [Flask](#flask)
 
 
 ### Android
 
 * [Android App Development Tutorial Malayalam](https://youtube.com/playlist?list=PLZ78Q1BKkdA1-eMVQOiBiMtQQb_vYWnvV) - Sabith Pkc Mnr
 
 
+### Bash/Shell
```
Comment (davorpa) [inline courses/free-courses-ml.md:27]:
```suggestion
### Bash and Shell
```

File: courses/free-courses-ml.md [hunk 3]
```diff
@@ -1,33 +1,113 @@
 ### Index
 
 * [Android](#android)
+* [Bash/Shell](#bash)
```
Comment (davorpa) [inline courses/free-courses-ml.md:4]: - Expand title of `bash/shell` section and fix its fragment id to get right cross linking.

```suggestion
* [Bash and Shell](#bash-and-shell)
```

File: courses/free-courses-ml.md [hunk 4]
```diff
@@ -1,33 +1,113 @@
...
+* [Bootstrap](#bootstrap)
+* [C](#c)
+* [C++](#cpp)
+* [Data Structure](#data-structure)
+* [Flutter](#flutter)
+* [Git](#git)
+* [HTML and CSS](html-and-css)
```
Comment (davorpa) [inline courses/free-courses-ml.md:11]: - Move `Bootstrap` as subsection of `HTML/CSS` (like in other listings)

```suggestion
* [C](#c)
* [C++](#cpp)
* [Data Structure](#data-structure)
* [Flutter](#flutter)
* [Git](#git)
* [HTML and CSS](html-and-css)
    * [Bootstrap](#bootstrap)
```

File: courses/free-courses-ml.md [hunk 5]
```diff
@@ -1,33 +1,113 @@
...
+### Git
+
+* [Git Tutorial Malayalam | Best Malayalam Git Tutorial | Date With Git](https://youtube.com/playlist?list=PLY-ecO2csVHdLhAO6TERaMJXP8aqyWVt-) - Brototype Malayalam
+
+
+### HTML and CSS
+
+* [CSS Malayalam Tutorial Full Course](https://youtube.com/playlist?list=PL1JrLEBAapUVvE-oCkKD5QhGG8nb0hhZk) - Yes Tech Media
+* [HTML Tutorial Malayalam Full](https://youtube.com/playlist?list=PL1JrLEBAapUVyMVQp6SmpytH1HPeCiNPH) - Yes Tech Media
+
```
Comment (davorpa) [inline courses/free-courses-ml.md:67]: Move `Bootstrap` section to `HTML / CSS` like in other listings

```suggestion


#### Bootstrap

* [BOOTSTRAP Malayalam Tutorial](https://youtube.com/playlist?list=PL1JrLEBAapUWqs_HbcYngAOmpPbiccqNy) - Yes Tech Media

```

File: courses/free-courses-ml.md [hunk 6]
```diff
@@ -1,33 +1,113 @@
...
+* [Bash Shell Scripting Malayalam Tutorial](https://youtube.com/playlist?list=PL1JrLEBAapUUcV6LES8bTEHJAOlkhmOHO) - Yes Tech Media
+
+
+### Bootstrap
+
+* [BOOTSTRAP Malayalam Tutorial](https://youtube.com/playlist?list=PL1JrLEBAapUWqs_HbcYngAOmpPbiccqNy) - Yes Tech Media
+
+
+### C
```
Comment (davorpa) [inline courses/free-courses-ml.md:37]: Move `Bootstrap` section to `HTML / CSS` like in other listings

```suggestion
### C
```

File: courses/free-courses-ml.md [hunk 7]
```diff
@@ -1,33 +1,113 @@
...
+
+
+### Flutter
+
+* [Flutter Malayalam Tutorial Series](https://youtube.com/playlist?list=PLY-ecO2csVHcUlBVvIMAa3dbja12TFJiN) - Brototype Malayalam
+
+
+### Git
+
+* [Git Tutorial Malayalam | Best Malayalam Git Tutorial | Date With Git](https://youtube.com/playlist?list=PLY-ecO2csVHdLhAO6TERaMJXP8aqyWVt-) - Brototype Malayalam
```
Comment (davorpa) [inline courses/free-courses-ml.md:60]: - Escape pipes on Markdown (see #5176 for more info)

```suggestion
* [Git Tutorial Malayalam \| Best Malayalam Git Tutorial \| Date With Git](https://youtube.com/playlist?list=PLY-ecO2csVHdLhAO6TERaMJXP8aqyWVt-) - Brototype Malayalam
```

File: courses/free-courses-ml.md [hunk 8]
```diff
@@ -1,33 +1,113 @@
...
+* [Java Programming Malayalam Tutorial](https://youtube.com/playlist?list=PL1JrLEBAapUXndScHzvMaSVOspebvFOsH) - Yes Tech Media
+
+
 ### JavaScript
 
 * [JavaScript Malayalam Tutorial](https://www.youtube.com/watch?v=3mjwtu4_0uk) - Yes Tech Media
+* [javascript tutorial in malayalam](https://youtube.com/playlist?list=PLBKJGko2JEdF4irCbI5BdHIEfxEdfMNqA) - Tintu Vlogger
+
+
+#### NodeJS
```
Comment (davorpa) [inline courses/free-courses-ml.md:80]: - Fix `node.js` title like in other listings

```suggestion
#### Node.js
```

File: courses/free-courses-ml.md [hunk 9]
```diff
@@ -1,33 +1,113 @@
...
 ### PHP
 
 * [PHP Programming Malayalam Tutorial for Beginners](https://www.youtube.com/watch?v=nFYWCouZ1UA) - Yes Tech Media
+* [php tutorial in malayalam](https://youtube.com/playlist?list=PLBKJGko2JEdH_T2ki6ty4xGV19qB7Hpmm) - Tintu Vlogger
 
 
 ### Python
 
 * [Python Programming Malayalam Tutorial](https://www.youtube.com/watch?v=ihnWXGPxNEk) - Yes Tech Media
+* [Python Tutorial Malayalam | Best Malayalam Python Programming Tutorial](https://youtube.com/playlist?list=PLY-ecO2csVHfbpOmWamlb8Mujjdnl1jks) - Brototype Malayalam
```
Comment (davorpa) [inline courses/free-courses-ml.md:102]: - Escape pipes on Markdown (see #5176 for more info)

```suggestion
* [Python Tutorial Malayalam \| Best Malayalam Python Programming Tutorial](https://youtube.com/playlist?list=PLY-ecO2csVHfbpOmWamlb8Mujjdnl1jks) - Brototype Malayalam 
```

**Review by davorpa — CHANGES_REQUESTED** [review]
Nice resources but needs some fixes to adapt them as same as in other listings

**Review by davorpa — APPROVED** [review]
LGTM after apply suggestions

**Comment by GH05T-HUNTER5** [comment]
> Nice resources but needs some fixes to adapt them as same as in other listings

Please Update

**Review by davorpa — APPROVED** [review]
Remains LGTM. Commit bca2f468692469ee0095cd7c941128e8ab299f09 doen't introduce changes

File: courses/free-courses-ml.md [hunk 10]
```diff
@@ -1,33 +1,113 @@
 ### Index
 
 * [Android](#android)
+* [Bash and Shell](#bash-and-shell)
+* [C](#c)
+* [C++](#cpp)
+* [Data Structure](#data-structure)
+* [Flutter](#flutter)
+* [Git](#git)
+* [HTML and CSS](html-and-css)
```
Comment (davorpa) [inline courses/free-courses-ml.md:10]: Fix TOC entry fragment of `HTML / CSS`

```suggestion
* [HTML and CSS](#html-and-css)
```

**Review by davorpa — APPROVED** [review]

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**